policy/store.go

```diff
@@ -26,6 +26,33 @@ func StoreFromDataFiles(paths []string) (storage.Store, error) {
 	return store, nil
 }
 
+// ReadDataFiles returns the paths to all data documents
+// at the given path. Any JSON or YAML document
+// could be a valid document.
+func ReadDataFiles(path string) ([]string, error) {
+	var data []string
+	err := filepath.Walk(path, func(currentPath string, info os.FileInfo, err error) error {
+		if err != nil {
+			return fmt.Errorf("walk path: %w", err)
+		}
+
+		if info.IsDir() {
```

Comment:
nit: the `.git` dir is pretty common when using conftest, there's probably value in using `filepath.SkipDir` if the path is .git'. We'll still want the `return nil`, as SkipDir will ignore the entire directory, which is desired for certain directories.

```go
if fileInfo.IsDir() && fileInfo.Name() == ".git" {
  return filepath.SkipDir
}
```